api/class.tx_ameosformidable.php

```diff
@@ -2233,7 +2241,7 @@ public function _render()
                 $this->getJSLoader()->includeBaseLibraries();
 
                 // the datahandler is executed
-                $sDH = $this->getDataHandler()->_doTheMagic(true);
```

Comment:
the $sDH variable is used in line 2587. I think it's not safe to remove the assignment here and in the following changes

Reply:
I'll have a look at it. We probably can just set it to `null` (as the `void` result is just `null`).

Reply:
Fixed, rebased and repushed. (I've changed `doTheMagic` to always return a string now.)